lotus/sem_ops/sem_filter.py

```
@@ -102,18 +105,34 @@ def sem_filter(
         )
         lotus.logger.debug(f"input to model: {prompt}")
         inputs.append(prompt)
-    kwargs: dict[str, Any] = {"logprobs": logprobs}
 
     if safe_mode:
-        estimated_total_calls = len(docs)
-        estimated_total_cost = sum(model.count_tokens(input) for input in inputs)
+        estimated_total_calls = len(docs) * max(1, n_sample)
```

we should raise a value error if n_sample is <= 0 since that does not make sense